test-openjdk/src/test/java/com/redhat/qe/openjdk/maven/MavenJavaArgsTest.java

```diff
@@ -180,7 +180,7 @@ public void javaClassPathTest() {
 	@Test
 	public void javaDiagnosticsTest() {
 		deploy(JavaS2IBuild.DOCKER_IMAGE_TEST_APP.getManagedBuild(), Collections.singletonMap("JAVA_DIAGNOSTICS", "true"));
-		if (OpenJDKTestConfig.isOpenJDK17()) {
+		if ( OpenJDKTestConfig.isOpenJDK21() ||OpenJDKTestConfig.isOpenJDK17()) {
```

Comment:
why there are configs/ifs for 17 and 21, an not for 11 and 8?

Reply:
JDK 11 is captured in line 187 and then jdk 8 line 191.

Reply:
ty!